tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1793,8 +1794,8 @@ private static Object userCanAccessForumWhy(User u, CLDRLocale locale) {
             return null; // Admin can modify all
         if (userIsTC(u))
             return null; // TC can modify all
-        if (locale.getLanguage().equals("und")) { // all user accounts can write
-            // to und.
+        if (SpecialLocales.getType(locale) == SpecialLocales.Type.scratch) {
```

Comment:
we actually could have multiple sandbox locales… i.e. different scripts, etc… without needing to revisit this code.